momentGW/pbc/kpts.py

```diff
@@ -348,9 +348,16 @@ def member(self, kpt):
         index : int
             Index of the k-point.
         """
-        if kpt not in self:
+        kpts2 = self.cell.make_kpts(self.kmesh)
+        kpts2 = self.wrap_around(kpts2)
+        kpt2_hash = {self.hash_kpts(kpt2): k for k, kpt2 in enumerate(kpts2)}
+
+        if kpt not in kpts2 and kpt not in self:
             raise ValueError(f"{kpt} is not in list")
-        return self._kpts_hash[self.hash_kpts(kpt)]
+        if kpt in self:
+            return self._kpts_hash[self.hash_kpts(kpt)]
+        else:
+            return kpt2_hash[self.hash_kpts(kpt)]
```

Comment:
what's going on with these changes?

Reply:
These changes where due to situations where I would get say `[-0,-0,-0]` rather than `[0,0,0]`. Am guessing this was caused by something else as  I cannot replicate the issue any more. Have reverted the code to the previous version.